cli/app.go

```diff
@@ -618,6 +620,134 @@ var app = &cli.App{
 						},
 					},
 				},
+				{
+					Name:      "tag",
+					Usage:     "tag binary data by filter or ids",
+					UsageText: createUsageText("data tag", nil, true),
+					Subcommands: []*cli.Command{
+						{
+							Name:  "ids",
+							Usage: "adds or removes tags from binary data by file ids for a given org and location",
+							UsageText: createUsageText("data tag ids", []string{dataFlagAdditionalTags, dataFlagOrgID,
+								dataFlagLocationID, dataFlagFileIDs, dataFlagTagCommand}, false),
+							Flags: []cli.Flag{
+								&cli.StringSliceFlag{
+									Name:     dataFlagAdditionalTags,
+									Required: true,
+									Usage:    "comma separated tags to add/remove to the data",
+								},
+								&cli.StringFlag{
```

Comment:
Instead of having the action as a flag, what do you think about having subcommands `add` and `remove`, so that the full commands would be `viam data tag add ids`, `viam data tag remove ids`, `viam data tag add filter`? I think this has the added benefit of being consistent with datasets [[0](https://github.com/viamrobotics/rdk/blob/9ffa0eb990ae09651700ffeca7d94143d26612db/cli/app.go#L651)] and also not obfuscating the command (add/remove) as a flag, when it seems like it should be a command

Reply:
Great idea! Thansk!